main.css

```diff
@@ -0,0 +1,79 @@
+/* Default Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Main Styles */
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  background: #e5e5e5;
+  padding: 10px 25px;
+}
+
+header {
+  width: 100%;
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: sticky;
+  top: 0;
+  left: 0;
+}
+
+header h1 {
+  color: #6070ff;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+header .menu-icon {
+  width: 45px;
+  color: #6070ff;
+}
+
+/* Headline Styles */
+
+.headline {
+  display: flex;
+  height: calc(100vh - 48px);
```

Comment:
- [ ] **[REQUIRED CHANGES]** While you did a great job setting the height (I also love to use the `calc` formula), it is important for this activity to stick to the template provided. Instead of adding a height, could you change the `.headline` to have a `padding` of `114px 24px` as proposed?

If you click on the figma template that you want to build, you will be able to see its properties, like colors, padding, font sizes and so on:
![image](https://user-images.githubusercontent.com/47700024/138102350-32ec6a18-124d-453e-919e-5aeeccb120b3.png)

Reply:
Adjusted with padding